### ISSUE TITLE
Mobile Meta Tag

### ISSUE THREAD

**Issue by Mikeysax** [JR-Charlie/firehose-app-one#1]
On mobile things don't look as good as they could be.

One thing you could look into is to add the "mobile meta tag" to your app, so the responsive effect of Bootstrap actually shows up on Mobile.

Let me know in the comments once you figured that step out :)

**Comment by JR-Charlie** [comment]
I added `<meta name="viewport" content="width=device-width, initial-scale=1">` in the `<head>` tag of application.html.erb, and I updated master.css.scss to include `@-ms-viewport{
  width: device-width;
}` 

Things are looking better, but now it's possible to swipe left or right and scroll a tiny bit on the page. Would tweaking the value of initial-scale help fix that?

**Comment by Mikeysax** [comment]
Can you link the Heroku here?

**Comment by JR-Charlie** [comment]
Yup, sorry about that: [Splurty App](https://splurty-jr-charlie.herokuapp.com/)

**Comment by Mikeysax** [comment]
Interesting, I do not see what you are mentioning. I'm not entirely sure what you are seeing. Maybe a screenshot would help.

**Comment by JR-Charlie** [comment]
Sorry for the delayed response, I was out of town for a few days. I just double checked the web app on my phone (Samsung S4), and was unable to reproduce the scrolling behavior I saw before. It's possible I accidentally zoomed in a bit and was able to swipe to scroll because of that. Happy it's displaying normally now though!
